os/compression/Kconfig

```diff
@@ -29,4 +29,12 @@ config COMPRESSION_BLOCK_SIZE
 	---help---
 		Enter block size to use for compression of binary.
 
+config NUMBER_BLOCKS_CACHING
+	int "Number of Blocks for Caching"
+	default 60
+	range 2 100
```

Comment:
3-100?

Reply:
Checked with 2 blocks as well. Logic is working according to it.
Just need a head and a tail in the list.